src/components/StyledInputAmount.js

```diff
@@ -15,7 +15,7 @@ const StyledInputAmount = ({ currency, min, max, value, onChange, ...props }) =>
       name="amount"
       maxWidth="10em"
       type="number"
-      step="1"
```

Comment:
Uhm, how could we have it float and keep a step of 1?

Reply:
Have tried it, the browser prevents that.

![Screenshot from 2019-05-08 17-06-17](https://user-images.githubusercontent.com/15707013/57390205-b74ef300-71b3-11e9-98f3-e1e655db436b.png)

Reply:
Great to know. Wondering if it's better without steps then. What do you think?

No one is gonna use the UI part of it. What are the other benefits? Validation?

Reply:
The major benefit is validation, if we remove `step` attribute, the browsers will set it by default as `1`. I think it is recommended to set `step` to `0.01` when dealing with money.

https://stackoverflow.com/questions/19011861/is-there-a-float-input-type-in-html5

Reply:
Cool. Do we have a `min` of 1 now? What happens if you enter `0.25`?

Reply:
The input `min` attribute is set as `minimumAmount / 100` which some cases `minimumAmount = 100`, hence `min = 1` .

Reply:
Good to note that precision validation is correctly handled too

![image](https://user-images.githubusercontent.com/1556356/57446774-c8504080-7255-11e9-91f4-01e73faacddf.png)